foxpuppet/windows/browser/panel_ui/panel_ui.py

```diff
@@ -104,6 +108,17 @@ def open_private_window(self) -> None:
                 lambda _: set(self.selenium.window_handles) - initial_handles,
                 message="Private window did not open",
             )
+            new_private_window = (
+                set(self.selenium.window_handles) - initial_handles
+            ).pop()
+            self.selenium.switch_to.window(new_private_window)
+
+            from foxpuppet.windows.browser.window import BrowserWindow
+
+            new_window = BrowserWindow(self.selenium, new_private_window)
+
+            if not new_window.is_private:
+                raise ValueError("The new window is not private.")
```

Comment:
`ValueError` doesn't seem like the right choice here. Should we just raise an `Exception` with this message?